libraries/Microsoft.Bot.Schema/Teams/BotMeetingNotification.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Specifies Bot meeting notification including meeting notification value.
+    /// </summary>
+    /// <typeparam name="T">The first generic type parameter.</typeparam>.
+    public abstract class BotMeetingNotification<T> : BotMeetingNotificationBase
```

Comment:
Just for my own curiosity, did you see a warning ((https://rules.sonarsource.com/csharp/RSPEC-1694) in Visual Studio for using the abstract keyword for a class that does not have concrete methods?

Reply:
I didn't see the warning in Visual Studio but thanks for pointing it out. Seems it's better to update it to a concrete class with the protected instructor, which also aligns with SMBA class model.
Will update it in the next iteration.